### DISCUSSION TITLE
第3节课作业(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#186] [作业]
第3节课作业(3班)

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/676751587

**Reply by hydexi0122** [reply]
有时间再用真实数据集测试一版

**Comment by 443752009** [comment]
http://t.csdnimg.cn/dGTHS

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/08/mmtutorial-3-assignment/

**Reply by SchweitzerGAO** [reply]
OpenXLab给点GPU吧😭😭😭

**Comment by zzmbhm** [comment]
https://docs.qq.com/doc/DZlJvc3NodExXemti

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/snlxv9wv129yadia?singleDoc# 《基于 InternLM 和 LangChain 搭建你的知识库作业》

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/oBnm4

**Comment by xiexiaoshinick** [comment]
[[笔记和作业]基于 InternLM 和 LangChain 搭建你的知识库 - 知乎 (zhihu.com)](https://zhuanlan.zhihu.com/p/677524722)

**Comment by wm901115nwpu** [comment]
https://zhuanlan.zhihu.com/p/676941539

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135578641

**Comment by semple030228** [comment]
https://blog.csdn.net/m0_49289284/article/details/135561304

**Comment by Gty-2002** [comment]
https://npumgk6rgc.feishu.cn/docx/L9W5dozOdorq7rx0l2kcvQ1WnEc?from=from_copylink 笔记及作业

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/677761013

**Comment by Sampson425** [comment]
作业
https://blog.csdn.net/m0_49908454/article/details/135581481
